pages/common/apktool.md

```diff
@@ -0,0 +1,16 @@
+# apktool
+
+> Reverse engineer APK files.
+> More information: <https://ibotpeaches.github.io/Apktool/>.
+
+- Decode an APK file:
+
+`apktool d {{file.apk}}`
+
+- Build a folder into an APK file:
+
+`apktool -b {{folder path}}`
```

Comment:
This should probably be changed to match the path token syntax.
```suggestion
`apktool -b {{path/to/directory}}`
```

Just interested, should this be `-b` or just `b`?
https://ibotpeaches.github.io/Apktool/documentation/#building

Reply:
Oh My! You're totally right, that was a huge typo.